ops/web-e2e.cloudbuild.yaml

```diff
@@ -46,7 +46,7 @@ steps:
   # Prepare Playwright test runner.
   # https://cloud.google.com/build/docs/optimize-builds/kaniko-cache
   - id: 'test-runner-build'
-    name: 'gcr.io/kaniko-project/executor:latest'
+    name: 'gcr.io/kaniko-project/executor:debug'
```

Comment:
question: Is the `debug` variant required to get verbose logging to work? It adds image size and expanded dependency footprint.